src/nylo/builtins.py

```diff
@@ -1,36 +1,60 @@
+# This file is a part of nylo
```

Comment:
builtins.py should not have the copyright. Only file beggining with __ shall have them.